app/views/groups/_add_member_modal.html.erb

```diff
@@ -38,10 +38,10 @@
           <i class="fas fa-external-link-alt"></i>
           <span><%= t("groups.invite_link_btn") %></span>
         <% end %>
-        <div class="collapse" id="group-invite-link-container">
-          <textarea readonly="readonly" id="embedInviteLink" class="group-invite-link-text" rows="3" cols="35" name="group-invite-link"></textarea>
-          <i class="fas fa-2x fa-regular fa-copy" onclick="copyToClipboard()" id="copyIcon" style="cursor:pointer"></i>
-        </div>
+        <div class="d-flex align-items-center">
```

Comment:
The design seems to have changed, also the copy button is overlapping with cross svg, please correct me if I am wrong. Could you please fix this and update the description with the image of fixed page.

<img width="498" alt="Screenshot 2025-02-03 at 7 14 21 PM" src="https://github.com/user-attachments/assets/a1360fbf-c28d-4b25-a171-37bf5e07e218" />